CMakeLists.txt

```diff
@@ -810,14 +822,9 @@ foreach(sourcefile ${C_TEST_EXES})
     add_test(NAME ${exename} COMMAND ${exename}${ARTIFACT_SUFFIX})
     add_dependencies(check ${exename}${ARTIFACT_SUFFIX})
 endforeach(sourcefile ${C_TEST_EXES})
-add_subdirectory(tools)
+endif()
 
-# Installation and packaging for Linux
-if(NOT WIN32)
```

Comment:
Based on my understanding, it seems like installation and packaging are applicable only to non-win32 platforms so far. With the current change, CMake will attempt to install binaries to some path on windows as well. So, I don't quite understand how the check for WIN32 can be removed here. Could you please explain more about his change? (I know there there was an earlier question you addressed around this, but I still don't get it). 

Also does `include(GNUInstallDirs)` work on Windows? What values do `CMAKE_INSTALL_LIBDIR` and `CMAKE_INSTALL_INCLUDEDIR` get?